src/components/Projects/TrendingProjectCard.tsx

```diff
@@ -0,0 +1,213 @@
+    <a
+      style={{
+        borderRadius: radii.lg,
+        cursor: 'pointer',
+        overflow: 'hidden',
+      }}
+      key={project?.handle}
+      href={`/#/p/${project?.handle}`}
+    >
+      {metadata ? (
+        <div style={cardStyle} className="clickable-border">
+          <div
+            style={{ marginRight: 20, display: 'flex', alignItems: 'center' }}
+          >
+            <div style={rankStyle}>{rank}</div>
+            <ProjectLogo
+              uri={metadata.logoUri}
+              name={metadata.name}
+              size={size === 'sm' ? 70 : 110}
+            />
+          </div>
+
+          <div
+            style={{
+              flex: 1,
+              minWidth: 0,
+              fontWeight: 400,
+            }}
+          >
+            <h2
+              style={{
+                color: colors.text.primary,
+                margin: 0,
+                overflow: 'hidden',
+                textOverflow: 'ellipsis',
+                fontSize: size === 'sm' ? 16 : 21,
+              }}
+            >
+              {metadata.name}
+            </h2>
+
+            {size === 'sm' ? null : (
+              <div>
+                <span style={{ color: colors.text.primary, fontWeight: 500 }}>
+                  @{project?.handle}
+                </span>
+                <span
+                  style={{
+                    marginLeft: 10,
+                    color: colors.text.tertiary,
+                    fontSize: '0.7rem',
+                    fontWeight: 500,
+                  }}
+                >
+                  V{terminalVersion}
+                </span>
+              </div>
+            )}
+
+            <div
+              style={{
+                color: colors.text.primary,
+                display: 'flex',
+                flexWrap: 'wrap',
+                width: '100%',
+              }}
+            >
+              <span
+                style={{
+                  display: 'flex',
+                  flexWrap: 'wrap',
+                  alignItems: 'baseline',
+                }}
+              >
+                <span style={{ fontWeight: 600 }}>
+                  <CurrencySymbol currency={CURRENCY_ETH} />
+                  {formatWad(project.trendingVolume, { precision })}{' '}
+                </span>
+                <span style={{ fontWeight: 500 }}>
+                  last {trendingWindowDays} days {''}
+                </span>
+                <span style={{ fontWeight: 600, color: colors.text.header }}>
+                  {percentGainText && <>{percentGainText}</>}
+                </span>
+              </span>
+            </div>
+
+            <div
+              style={{
+                fontWeight: 400,
+                color: colors.text.secondary,
+                fontSize: 13,
+                marginTop: 2,
+              }}
+            >
+              {formattedNum(project.trendingPaymentsCount)} payment
+              {project.trendingPaymentsCount > 1 ? 's' : ''}
```

Comment:
Not super important, but remember 0 values are plural too. In all my React codebases, I like to keep a little function handy:

```typescript
function pluralize<S, P>(count: number, singular: S, plural: P) {
  if (count == 1) {
    return singular
  }

  return plural
}
```

That way, this code could look like:

```suggestion
              {formattedNum(project.trendingPaymentsCount)}
              {pluralize(
                project.trendingPaymentsCount,
                ' payment',
                ' payments',
              )}
```

Reply:
Great suggestion. Added

Reply:
@torvusbug @johnnyd-eth ideally we'd leverage the lingui JS `plural` helper for plurals, this way we ensure we're generating nice translatable plurals: https://lingui.js.org/guides/plurals.html

Reply:
Yeah great idea @aeolianeth. I've changed the new `pluralize` function to incorporate i18n plural functionality